vsc-error.yml

```diff
@@ -34,22 +34,30 @@ enumerations:
         #
       - name: null
         value: 0
-        description: No return value
+        description: No return value. Shall be used only as default value, shall never be explicitly returned
+                     by a service implementation.
         
         #
         # Positive success codes
         # 
       - name: ok
         value: 1
-        description: No error.
+        description: Operation succeeded. Definition of success is method dependent.
+                     Depending on operation, it can either indicate that the request was accepted or that the
+                     corresponding physical operation (e.g. FOTA update, seat movement) has succeeded.
 
       - name: in_progress
         value: 2
-        description: The operation has been initialized and is in progress
+        description: The operation has been initialized and is in progress.
```

Comment:
I am a bit unsure if a service implementation ever will return "in_progress". For single-reply methods I assume no, as I assume that "in_progress" always shall be followed up by ok/completed/error.

If we support multi/stream-replies in the future we could possibly use "in_progress", but I am not sure, maybe we will not give an explicit error messages for individual stream replies at all, but rather just give an error code when we have completed or experienced a failure.